llama.cpp

```diff
@@ -570,19 +570,37 @@ static void ggml_graph_compute_helper(std::vector<uint8_t> & buf, ggml_cgraph *
 // llama helpers
 //
 
+static inline void * llama_host_malloc(size_t n) {
```

Comment:
`static inline` doesn't actually mean anything different from `static` in a `.cpp` file.

If you really want it to be inlined, you would use `__attribute__((always_inline))` (GCC/Clang) or `__forceinline` (MSVC).

Reply:
I removed static and keep the inline hint.

Reply:
You should actually remove 'inline' and keep 'static'. This function is internal to the .cpp file and doesn't need external linkage.

Reply:
Yes, you're correct; it reminds me of my long-neglected memory related to miscellaneous C linkage.

Updated